src/main/java/org/jabref/gui/preferences/journals/JournalAbbreviationsTabViewModel.java

```diff
@@ -230,7 +230,7 @@ public void addAbbreviation(Abbreviation abbreviationObject) {
     public void addAbbreviation() {
         addAbbreviation(new Abbreviation(
                 Localization.lang("Name"),
-                Localization.lang("Abbreviation"),
+                Localization.lang("abbreviation"),
```

Comment:
Please revert the change here

Reply:
Certainly! I think I accidentally made this change, sorry.